api/server/server.go

```diff
@@ -163,7 +150,7 @@ func (s *server) Dispatch() error {
 
 func (s *server) RegisterChain(chainName string, ctx *snow.ConsensusContext, vm common.VM) {
 	ctx.Lock.Lock()
-	handlers, err := vm.CreateHandlers(context.TODO())
+	handler, err := vm.NewHTTPHandler(context.TODO())
```

Comment:
We also did a rename, which I can also undo. But since we're already messing with this interface signature it seemed forgivable to try to sneak in.